src/main/java/org/broadinstitute/dsde/consent/ontology/datause/services/TextTranslationService.java

```diff
@@ -1,17 +1,21 @@
 package org.broadinstitute.dsde.consent.ontology.datause.services;
 
 import org.broadinstitute.dsde.consent.ontology.model.DataUseSummary;
+import org.broadinstitute.dsde.consent.ontology.model.Recommendation;
 
 import java.io.IOException;
+import java.util.HashMap;
 
 public interface TextTranslationService {
 
-    enum TranslateFor { DATASET, PURPOSE }
+    enum TranslateFor { DATASET, PURPOSE, PARAGRAPH }
 
     DataUseSummary translateDataUseSummary(String dataUseString);
 
     String translateDataset(String dataUse) throws IOException;
 
+    HashMap<String, Recommendation> translateParagraph(String paragraph) throws Exception;
```

Comment:
I think it's better to avoid implementation types like `HashMap` or `ArrayList` when possible, so I would use `Map` here.
```suggestion
    Map<String, Recommendation> translateParagraph(String paragraph) throws Exception;
```

Also, and I think I mentioned this before, I would avoid non-specific exception types like `Exception`.

Reply:
Thank you, yes, will do some refactoring here.